packages/app-democracy/src/Overview/Propose.tsx

```diff
@@ -3,40 +3,68 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { Call } from '@polkadot/types/interfaces';
-import { I18nProps } from '@polkadot/react-components/types';
 import { ApiProps } from '@polkadot/react-api/types';
 
 import BN from 'bn.js';
 import React from 'react';
-import { RouteComponentProps } from 'react-router';
-import { withRouter } from 'react-router-dom';
 import { createType } from '@polkadot/types';
-import { Button, Extrinsic, InputAddress, InputBalance, TxButton, TxComponent } from '@polkadot/react-components';
+import { Button, Extrinsic, InputAddress, InputBalance } from '@polkadot/react-components';
 import { withApi, withMulti } from '@polkadot/react-api';
 
-import translate from './translate';
+import translate from '../translate';
+import TxModal, { TxModalState, TxModalProps } from '@polkadot/react-components/TxModal';
```

Comment:
(nit) move this to just below the `'@polkadot/react-components'` imports on line 11

Reply:
Yep, sure. Didn't see the duplication of account cause didn't create one before testing :)
I've removed that input and adjusted imports